src/main/js/controls/Graph/helpers/helpers.js

```diff
@@ -675,6 +675,43 @@ const setAxisPadding = (axisPadding, { config }) => {
         axisPadding[yAxis] = config.axisPadding;
     }
 };
+/**
+ * Removes the No Data View from the node
+ *
+ * @param {d3.selection} svg - d3 selection node of svg.
+ * @returns {undefined} - returns nothing
+ */
+const removeNoDataView = (svg) => {
+    svg.select(".no-data-view").remove();
+};
+/**
+ * Append No Data View to the graph node.
+ *
+ * @param {object} config - config object derived from input JSON
+ * @param {d3.selection} svg - d3 selection node of svg.
+ * @returns {d3.selection} d3 selection node of svg.
+ */
+const drawNoDataView = (config, svg) => {
+    const yAxisPadding = config.height / 4;
```

Comment:
Can you move this magic number to `constants.js`? Also why 4?

Reply:
As per the requirement [here](https://wiki.cerner.com/display/UserExperience/Advanced+Graphing?preview=/1456055900/2063505555/NoData_Callouts_7.18.2019.pdf), 

> The middle 1/3 of the graph should have an opacity of 40% to call attention to the ‘No Data’ message.

I will update it to 3. Any suggestion what name can be given to the const?